Gems/Atom/RHI/Vulkan/Code/Source/RHI/DescriptorSet.h

```diff
@@ -39,7 +39,7 @@ namespace AZ
             using Base = RHI::DeviceObject;
             friend class DescriptorPool;
 
-            static constexpr size_t ViewsFixedsize = 16;
+            static constexpr size_t ViewsFixedsize = 2;
```

Comment:
Is this even used anywhere? Remove if not.

Reply:
This is still used in the remaining small_vector's in this file, although the small_vector's themselves aren't pre-allocated anymore